catalog/video/catalog-mpegts-stream/src/main/java/org/codice/alliance/video/stream/mpegts/UdpStreamMonitor.java

```diff
@@ -331,13 +345,21 @@ public void updateCallback(Map<String, Object> properties) {
                 return;
             }
 
+            if (properties.containsKey(METATYPE_DISTANCE_TOLERANCE) && properties.get(
```

Comment:
Shouldn't this just be `if (!checkMetaTypeClass(properties, METATYPE_DISTANCE_TOLERANCE, Double.class))`?